NEWS.md

```diff
@@ -1,10 +1,72 @@
 # Development
 
-## New features
+## New features and changes
+
+- The new `.include` argument of `config_log()`, `run_log()`, and
+  `summary_log()` limits the result to the specified run names or tags.  (#484,
+  #526)
+
+- Models can now be starred.  See `add_star()` and `remove_star()`.  (#487)
+
+- New convenience functions `get_omega()`, `get_sigma()`, and `get_theta()`
+  return labeled values, with the `OMEGA` and `SIGMA` values expanded to a full
+  matrix.  (#515)
+
+- `param_estimates` gained an `.alpha` argument that's useful for identifying
+  the ETAs flagged by the `eta_pval_significant` heuristic.  (#497)
+
+- `nm_join()` now aborts if the join column has duplicate values, suggesting to
+  the caller that the table format probably needs to be widened to prevent
+  NONMEM from truncating the values.  (#533)
+
+- `nm_file()` and friends now detect duplicate column names and make them
+  unique.  (#530, #539)
```

Comment:
Should this be classified as a bug fix? Im ok to leave it here, but previously `nm_file()` would error if there were duplicate column names in the input file

Reply:
I suppose it depends on whether it's actually a bad file being fed in that bbr is being nice dealing with or if it's an expected case that we should handle.

But I don't think it matters much one way or the other, so I'll move it to the bugs section.

Reply:
Yeah thats a fair point. Entirely up to you then